ployst/github/hooker.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+import hashlib
+import hmac
+import json
+
+
+DEBUG = True
+
+
+class GithubHookException(Exception):
+    pass
+
+
+def compute_github_signature(body, secret):
+    return hmac.new(secret, body, hashlib.sha1).hexdigest()
+
+
+class GithubHookHandler(object):
+    """
+    A general purpose-class to help build github hook handlers.
+
+    See https://developer.github.com/webhooks/ for Github's webhook spec.
+
+    NOTE: This module may be spin-off into a separate library.
+    """
+
+    @staticmethod
+    def get_org_repo(request):
```

Comment:
I don't like this method much, and the partial duplication with `org_repo` below. The reason for this to exist is that we generate github secrets tailored to each repo (they include org and repo name). 

I'm not sure what the real reason behind that is, but besides the currently ugly code it poses one additional potential issue: in the future we may want to support organisation webhooks, like repo creation events (https://developer.github.com/v3/activity/events/types/#repositoryevent) and let ployst know when you have created a new repo, e.g. to suggest you to include it in a project.

What do you think @alexcouper ?